invenio_records_permissions/policies/records.py

```diff
@@ -64,11 +64,11 @@ class RecordPermissionPolicy(BasePermissionPolicy):
 def get_record_permission_policy():
     """Return RecordPermissionPolicy.
 
-    Relies on ``RECORDS_PERMISSIONS_RECORD_PERMISSION_FACTORY`` to
+    Relies on ``RECORDS_PERMISSIONS_RECORD_PERMISSION_POLICY`` to
```

Comment:
Minor: Isn't this ``RECORDS_PERMISSIONS_RECORD_POLICY``?

Reply:
Sure, we can make it shorter!